.github/workflows/greetings.yml

```
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Welcome! Thank you for opening your first issue!'
```

```suggestion
        issue-message: 'Welcome! Thank you for opening your first issue! If you want to try tackling it, open a PR.'
```
We might get more PRs if we encourage it.